### PR TITLE
husky formatter

### PR DESCRIPTION
**Pull request by chewnoill**
* formats js, jsx, json files

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -306,4 +306,4 @@ type WidgetCollection {
   byIdOrFirst(id: Int): Widget!
   enabled: [Widget!]!
   next(currentWidgetId: Int): Widget!
-}
```
Comment (cbarber) [inline schema.graphql:309]: This will be problematic since our `graphql:schema:dump` CI step will likely fail on this

**Comment by chewnoill** [comment]
> reasons mentioned in slack

The only reason I saw was "You'll have unrelated prettier changes on your last commit if you have more than one", 
which is solved with [git-format-staged](https://github.com/hallettj/git-format-staged), because it only formats staged hunks. 

Are there other reasons as well?

**Comment by cbarber** [comment]
Yes, don't impose workflow choices on other developers

File: schema.graphql (hunk 1 above)
Comment (chewnoill) [reply schema.graphql]: dropped gql files from the glob pattern

**Pull request merged**